pkg/agent/flowexporter/connections/conntrack_test.go

```diff
@@ -76,28 +82,59 @@ func TestConnTrackSystem_DumpFlows(t *testing.T) {
 	}
 	// Test the DumpFlows implementation of connTrackSystem
 	mockNetlinkCT := connectionstest.NewMockNetFilterConnTrack(ctrl)
-	connDumperDPSystem := &connTrackSystem{
-		nodeConfig,
-		serviceCIDR,
-		mockNetlinkCT,
-	}
+	connDumperDPSystem, acctErr, timestampErr := NewConnTrackSystem(nodeConfig, serviceCIDR)
+	connDumperDPSystem.connTrack = mockNetlinkCT
 	// Set expects for mocks
 	mockNetlinkCT.EXPECT().Dial().Return(nil)
 	mockNetlinkCT.EXPECT().DumpFilter(conntrack.Filter{}).Return(testFlows, nil)
 
-	conns, err := connDumperDPSystem.DumpFlows(openflow.CtZone)
+	conns, conntrackMetrics, err := connDumperDPSystem.DumpFlows(openflow.CtZone)
 	if err != nil {
 		t.Errorf("Dump flows function returned error: %v", err)
 	}
 	assert.Equal(t, 1, len(conns), "number of filtered connections should be equal")
+
+	if acctErr == nil {
+		conntrackAcct, err := sysctl.GetSysctlNet("netfilter/nf_conntrack_acct")
```

Comment:
@dreamtalen @srikartati Sorry I missed that earlier. A unit test should ideally not rely on platform-specific constructs. Is it possible to move this to integration tests? Or define an interface to access that information?

Reply:
@antoninbas I made this platform-dependent by adding // +build linux on top as we only support linux for now. Took the cue from pkg/agent/route/route_windows_test.go

I see the point of making the unit tests independent of platforms. 
For now, could we change the name of file like above: conntrack_linux_test.go?
When the windows support comes in the imminent future, we modify the file with interfaces for sysctl calls that works for all platforms to make these unit tests platform-independent in future PR?